components/form/builder/src/index.js

```diff
@@ -162,5 +162,9 @@ export {fieldSizes as formBuilderFieldSizes}
 export {pickFieldById as formBuilderPickFieldById}
 export {changeFieldById as formBuilderChangeFieldById}
 export {fieldsNamesInOrderOfDefinition as formBuilderFieldsNamesInOrderOfDefinition}
-export {checkConstrainstsFactory} from './Standard'
+export {
+  checkConstraintsFactory,
+  // WARNING do not use the following export, it is kept for backwards compatibility
+  checkConstraintsFactory as checkConstrainstsFactory
```

Comment:
I think it's more interesting if you do a function that only calls the right one inside but first puts a warning on the console that it's deprecated. You can do that so that the console is only seen if the environment is not Production

Reply:
how about this https://github.com/SUI-Components/adevinta-spain-components/pull/253/commits/f2555aab5f6849435c7aaa5e63476c547922e917 ?